### DISCUSSION TITLE
Anyway to externally connect to extension to send a message?

### DISCUSSION THREAD

**Discussion by ddeisadze** [wxt-dev/wxt#461] [Q&A] [answered]
I want to use externally_connectable in MV3, i dont see a whole lot of documentation.

Here is the use case: I want to login from my webapp, and send the auth message (and a token) to the background script.

**Comment by aklinker1** [comment, marked as answer]
WXT doesn't provide any custom APIs or helpers for this, you'll have to use the standard API. 

Here's Chrome's docs for sending messages from your web-app to your extension:
https://developer.chrome.com/docs/extensions/develop/concepts/messaging#external-webpage
